pkgs/top-level/all-packages.nix

```
@@ -26107,6 +26107,8 @@ with pkgs;
 
   faircamp = callPackage ../applications/misc/faircamp { };
 
+  famistudio-bin = callPackage ../applications/audio/famistudio/bin.nix { };
```

```suggestion
  famistudio = callPackage ../applications/audio/famistudio { };
```

Please use https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#source-provenance-sec-meta-sourceprovenance instead.

---

Done.